content/company/strategy/code-graph/batch-changes/index.md

```diff
@@ -14,29 +14,19 @@ Batch Changes is a tool to find code that needs to be changed and change it at s
 
 ## Vision
 
-### 1-year vision
```

Comment:
Perhaps instead of completely omitting this section, you can leave it in but only link to "Themes" down below? It just feels like I missed something when the first time frame I see on this page is on the scale of 3 years haha.